queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE neutered AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name LIKE 'Agumon' OR name LIKE 'Pikachu';
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered;
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] Nice job on implementing these queries however I can't find one query that should perform this task in the database:
`it can find all animals not named Gabumon.`